src/main/java/org/gridsuite/sensitivityanalysis/server/dto/ContingencyListExportResult.java

```diff
@@ -15,7 +15,7 @@
 import java.util.List;
 import java.util.UUID;
 
-/*
+/***
```

Comment:
this is still not the correct header ? there are only two '*'

Reply:
done